Shared/View/CreateActivityView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  CreateActivityView.swift
+//  Fun Gen
+//
+//  Created by Elena on 5/2/22.
+//
+
+import SwiftUI
+
+struct CreateActivityView: View {
+    @State private var selectedCategory: Category = .outdoor
+
+    @State var optionList: [String] = []
+    @State var newOption = ""
+    
+    @State var friendList: [String] = []
+    @State var newFriend = ""
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text("Create Activity View").padding()
+            Text("Input Activity").font(.title).padding()
+            TextField("Enter in activity title", text: .constant("")).padding()
+            HStack {
+                Text("Select Category:")
+                // Loop through the category enum in Category.swift using a picker
+                Picker("Category", selection: $selectedCategory) {
+                    ForEach(Category.allCases, id: \.self) { category in
+                        Text(category.rawValue.capitalized)
+                    }
+                }
+            }.padding()
+            
+            Group {
+                Text("Options")
+                    .font(.title).padding()
+               ForEach(optionList, id: \.self) { item in
+                   Text(item).padding(.leading).padding(.bottom, 4)
+               }
+               TextField("Suggest an option", text: $newOption) {
+                   self.optionList.append(self.newOption)
+                   self.newOption = ""
+               }.padding()
+           }
```

Comment:
Make sure this is indented to the same level as Group, as requested by SwiftLint. Alternatively, select all the code and use <kbd>control</kbd> + <kbd>i</kbd> to let Xcode format this for you